Process.c

```diff
@@ -133,12 +132,11 @@ void Process_printBytes(RichString* str, unsigned long long number, bool colorin
       number %= 1000;
       len = xSnprintf(buffer, sizeof(buffer), "%03lluG ", number);
       RichString_appendnAscii(str, processGigabytesColor, buffer, len);
-   } else if (number < 100000 * ONE_M) {
+   } else if (number < 100 * ONE_G) {
```

Comment:
Wait a minute. Do we need type casting here?
```suggestion
   } else if (number < 100ULL * ONE_G) {
```

Reply:
Likely, yes. I thought all ONE_x were ULL but it turns out ONE_K, ONE_M and ONE_G are UL whereas ONE_T and ONE_P are indeed ULL.